annotations/quadriga.py

```diff
@@ -275,3 +257,150 @@ def parse_response(raw_response):
         tag = child.tag.replace(QDNS, '')
         data[tag] = child.text
     return data
+
+
+
+def build_concept_node(concept, user, creation_time, source_uri):
+    """
+    Helper function to build a concept node dictionary.
+    """
+    # Get all appellations that created this concept and add stringRep (annotated text)
+    appellations = Appellation.objects.filter(interpretation=concept)
+    
+    term_parts = []
+    for appellation in appellations:
+        term_parts.append({
+            "position": appellation.startPos if appellation.startPos else 0,
+            "expression": appellation.stringRep or "",
+            "normalization": "",
+            "formattedPointer": "",
+            "format": ""
+        })
```

Comment:
I'm confused, I don't think this is correct. per appellation there might be more than one term part, but those term parts are not coming from more than one appellation. 
appellation -has -> concept
appellation -has -> term parts

but not all term parts for all appellations should be collapsed into one node. Does that make sense?